docs/ref/models/querysets.txt

```diff
@@ -1903,6 +1903,29 @@ SQL equivalent:
 
     SELECT ... WHERE x=1 OR y=2
 
+XOR (``^``)
+~~~~~~~~~~~
+
+.. versionadded:: 4.0
+
+Combines two ``QuerySet``\s using the SQL ``XOR`` operator.
+
+The following are equivalent::
+
+    Model.objects.filter(x=1) ^ Model.objects.filter(y=2)
+    from django.db.models import Q
+    Model.objects.filter(Q(x=1) ^ Q(y=2))
+
+SQL equivalent:
+
+.. code-block:: sql
+
+    SELECT ... WHERE x=1 XOR y=2
+
+.. note::
+    ``XOR`` is emulated in backends that don't support it by changing
+    ``x=1 XOR y=2`` into ``(x=1 OR x=2) AND NOT (x=1 AND y=2)``.
```

Comment:
Lets add a blank line after the directive and also make it clear which backends currently support this:
```suggestion
.. note::

    ``XOR`` is emulated in backends that don't support it by changing
    ``x=1 XOR y=2`` into ``(x=1 OR x=2) AND NOT (x=1 AND y=2)``. Currently
    ``XOR`` is only supported by MySQL and MariaDB.
```

Reply:
I have added this.

I only see/added backend settings for MySQL though, and I'm not sure if MariaDB uses those settings?